plugins/seccomp/src/PoliciesTable.tsx

```diff
@@ -0,0 +1,372 @@
+    setIsConfirmDialogOpen(true);
+  }
+
+  function handlePolicyMerge() {
+    setIsConfirmDialogOpen(true);
+  }
+
+  function handleCheckBox(e: any, data: any) {
+    const policies = selectedPolicies ? [...selectedPolicies] : [];
+    if (e.target.checked) {
+      policies.push(data);
+    } else {
+      policies.splice(policies.indexOf(data), 1);
+    }
+    setSelectedPolicies(policies);
+  }
+
+  function handleConfirm() {
+    const standalonePods = policiesToConsider
+      .filter((policy: any) => policy?.rootWorkload === 'Pod')
+      .map((policy: any) => policy?.metadata.name);
+    const massagedPoliciesToConsider = policiesToConsider.filter(
+      (policy: any) => policy?.rootWorkload !== 'Pod'
+    );
+    massagedPoliciesToConsider.forEach((policy: any) => {
+      applySeccompProfileToWorkload(policy.rootWorkloadName, namespace, policy.rootWorkload, {
+        type: 'Localhost',
+        profile: policy.status.localhostProfile,
+      })
+        .then(() => {
+          enqueueSnackbar(`policy ${policy?.metadata?.name} successfully applied`, {
+            variant: 'success',
+          });
+          setSelectedPolicies(null);
+          callback(policy);
+        })
+        .catch((error: Error) => {
+          console.log(error);
+        });
+    });
+    if (standalonePods && standalonePods.length !== 0) {
+      enqueueSnackbar(
+        `policies ${standalonePods.join(
+          ', '
+        )} couldn't be applied as they do not belong to a workload`,
+        {
+          action: key => (
+            <Button onClick={() => closeSnackbar(key)} color="secondary">
+              Ok
+            </Button>
+          ),
+          persist: true,
+        }
+      );
+    }
+    setSelectedPolicies(null);
+  }
+
+  return (
+    <Box py={2}>
+      <ConfirmDialog
+        open={isConfirmDialogOpen}
+        description={generateDialogDescription()}
+        onConfirm={handleConfirm}
+        handleClose={() => {
+          setIsConfirmDialogOpen(false);
+          setSelectedPolicies(null);
+        }}
+      />
+      {selectedPolicies?.length > 0 && (
+        <Grid container spacing={2}>
+          {shouldHaveApply && (
+            <Grid item>
+              <Button variant="outlined" onClick={handlePolicyApply}>
+                <Icon icon={checkIcon} width="20" height="20" />
+                <Typography>Apply</Typography>
+              </Button>
+            </Grid>
+          )}
+          {/* The merge functionality still needs to be completed */}
+          {/* <Grid item>
+          <Button variant="outlined" onClick={handlePolicyMerge}>
+            <Typography>Merge</Typography>
+          </Button>
+        </Grid> */}
+          <Grid item>
+            <Button
+              variant="outlined"
+              onClick={() => handlePolicyDelete(selectedPolicies, () => setSelectedPolicies(null))}
+            >
+              <Typography>Delete</Typography>
+            </Button>
```

Comment:
This should use a delete icon instead, but more importantly, be shown within its section (right now it's on top of the section):
![image](https://user-images.githubusercontent.com/1029635/151412418-59895fe4-dc26-4b33-902e-be5a5c4b0b72.png)